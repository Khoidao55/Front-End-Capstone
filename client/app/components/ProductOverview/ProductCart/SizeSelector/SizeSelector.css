.product-dropdown-box {
  outline: none;
  text-decoration: none;
  border-radius: 200px;
  padding: 5px;
  transition: background-color 0.2s, border 0.2s, color 0.2s;
}