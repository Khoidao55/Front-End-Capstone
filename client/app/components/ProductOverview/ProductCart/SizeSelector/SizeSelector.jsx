import React from 'react';

const SizeSelector = ({ onSelectSize, arraySkus }) => {
  if (arraySkus !== undefined) {
    return (
      <div>
        <select onChange={onSelectSize}>
          <option>Select A Size</option>
          {arraySkus.map((skus) => {
            return (
              <option key={arraySkus.indexOf(skus)} value={skus.size}>{skus.size}</option>
            )
          })}
        </select>
      </div>
    );
  }
};

export default SizeSelector;
