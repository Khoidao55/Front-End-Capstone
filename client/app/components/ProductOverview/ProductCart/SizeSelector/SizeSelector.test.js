import React from 'react';
import { shallow } from 'enzyme';
import SizeSelector from './SizeSelector.jsx';

describe('Size Selectore', () => {
  test('should render my component', () => {
    const wrapper = shallow(<SizeSelector />);

    expect(wrapper).toMatchSnapshot();
  });
});
