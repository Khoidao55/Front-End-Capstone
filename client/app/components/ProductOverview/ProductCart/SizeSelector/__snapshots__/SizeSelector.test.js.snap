// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Size Selectore should render my component 1`] = `ShallowWrapper {}`;
